### DISCUSSION TITLE
online.db > youtube_preview

### DISCUSSION THREAD

**Discussion by MeuhMeuh62** [ppy/osu#27434] [Q&A] [answered]
Hello, i have a question about `online.db` into osu folders.
Let's open it:
![image](https://github.com/ppy/osu/assets/86471514/78dc0649-3e07-4a83-a078-4cac1df577f9)
there are a header called `youtube_preview` which have youtube ID videos (i suposed)
I try to see if videos are available but nope
![image](https://github.com/ppy/osu/assets/86471514/e2cd188b-d9ed-4d96-a99d-431ff1ac0a85)
All bm which comport a youtube video ID other than `NULL` got a `last_update` into 2014-05-15 to 2014-05-18

I check a bm on osu website i picked that one for an example :
`beatmap_id` : 169089`   last_update` : 2014-05-18 17:20:31+00:00   `youtube_preview` : GLejJaLpO1M
![image](https://github.com/ppy/osu/assets/86471514/da4e2276-8b8f-4219-8cd2-ce907be658b9)
![image](https://github.com/ppy/osu/assets/86471514/99cee7a3-ab6b-4293-87e2-24120aaa657f)
I don't understand why on website it's written ranked the 6 august 2012 and it's different on the file...
Is it online.db corrupted ?

**Comment by bdach** [comment, marked as answer]
The videos were displayed on the old website ([example in question](https://old.ppy.sh/s/55799)). Consider that deprecated and unsupported functionality.

> I don't understand why on website it's written ranked the 6 august 2012 and it's different on the file...

`last_updated` can change after rank due to a variety of internal reasons. The last update date changes to let clients know the change happened.

> Is it online.db corrupted ?

Nothing of what you've pointed out here suggests this is the case.
